src/js/stores/NodeHealthStore.js

```diff
@@ -94,7 +93,7 @@ const NodeHealthStore = Store.createStore({
   },
 
   shouldPoll: function () {
-    return !_.isEmpty(this.listeners(HEALTH_NODES_CHANGE));
+    return (this.listeners(HEALTH_NODES_CHANGE).length === 0);
```

Comment:
wait, isn't the original code checking if its *not* empty, and this checks if it *is* empty?